generators/entity/index.js

```diff
@@ -207,6 +207,7 @@ module.exports = EntityGenerator.extend({
         this.changelogDate = this.fileData.changelogDate;
         this.dto = this.fileData.dto;
         this.service = this.fileData.service;
+        this.fluentMethods = this.fileData.fluentMethods;
```

Comment:
may be you can add `--fluentMethods` as an option to sub generator as well when creating entities by hand.
dont forget to PR the JDL project as well